### ISSUE TITLE
Super slow training when dealing with .h5 training data 

### ISSUE THREAD

**Issue by Aslizy** [MVIG-SJTU/AlphaPose#428]
Hi, 

I am running training experiments using the pytorch version of alphapose. Unlike following the data structure like the author (put images in a directory and put image path and annotation in another json, load the images on the fly during training), I am using some synthetic data for training (120k image patches) and I created a .h5 file including the RGB information of the image patches, annotated heatmap and mask. The resolution I am using is (320, 160). I am using batchsize 192 over 3GPUs. The training is super slow. It took 4 days to reach 4 epoch. 

In comet.ml, I found out the GPU memory usage is high, and the GPU utilization is quite sparsely used. 

I am out of clue at this moment. I would really appreciate if anyone could provide some ideas or some suggestions here. 

Btw, I am Chinese. You can post in Chinese if you prefer.

**Comment by Fang-Haoshu** [comment]
Hi! I am not sure whether h5 supports RGB info well or not.. Can you try reading the images using opencv, like we did? I guess IO is the bottleneck
